server/mldb_server.cc

```diff
@@ -186,11 +186,15 @@ initRoutes()
 
    // MLDB-1380 - make sure that the CPU support the minimal instruction sets
     if (supportsSystemRequirements()) {
+        const auto queryStringDef = "The SQL query string. Must be defined "
```

Comment:
Pour éviter la confusion entre query string et query string, je dirais: The string representing the SQL query. Must be ...